docs/understanding-the-flutterflow-ui/projects/how-to-collaborate-on-projects.md

```diff
@@ -0,0 +1,77 @@
+---
+slug: how-to-collaborate-on-projects
+title: How to Collaborate on Projects
+tags: []
+toc_max_heading_level: 5
+---
+FlutterFlow allows for multiple ways to collaborate on a project as well as the ability to manage the collaboration through permissions.
+
+## Sharing a Project with a User
+
+You can also invite users to your project who are not part of your organization. For instance, you might want to share your work with clients, stakeholders, or team members of the client.
+
+When you invite users to your project, you have the flexibility to assign them different roles depending on their level of involvement. For example, you can grant editing privileges to collaborators who are actively working on the project while assigning read-only access who need to view the progress. Alternatively, you can also make them the owner of the project, giving them full control.
```

Comment:
Shall we mention about the different roles and what they mean somewhere here?

Reply:
What do you mean? We have a couple different things that could be considered roles. 
1. What we call "Status" - Owner and Active
2. What we call "Read Only" - boolean values
3. Team Roles - Owner and Editor (I think)

Reply:
Oh I think I meant team roles because you mentioned this --- _you have the flexibility to assign them different roles depending on their level of involvement._ So i was pointing out to hint at the different roles one can assign to team members.

Reply:
Sorry, maybe I'm misunderstanding. But the text reads: "When you invite users to your project, you have the flexibility to assign them different roles depending on their level of involvement. For example, you can grant editing privileges to collaborators who are actively working on the project while assigning read-only access who need to view the progress." That is, right after we introduce "roles" we talk about editing privileges and read-only. Is that insufficient?

Also, those team roles would be in the sharing with Org section, right?

Reply:
I guess you are right, maybe i was thinking there were more roles lol. I overlooked it, nevermind, this is fine.